lib/src/data.dart

```diff
@@ -0,0 +1,14 @@
+// part of '../animated_icon_button.dart';
+import 'package:flutter/material.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'data.freezed.dart';
+
+@freezed
+abstract class AnimatedIconButtonEntry with _$AnimatedIconButtonEntry {
```

Comment:
Can we simplify this name ?
It looks very large and uncomfortable.

Something like IconItem ... or somehow different ?

Reply:
If you like to decide a name that does not occur in any other package, I'll gladly take it

Reply:
What do you think about IconItem ? 
Do you know about the use of such a name?

Reply:
I personally prefer AnimatedIconItem, because it shows the belonging to the library.
And I checked, neither IconItem or AnimatedIconItem is taken.